R/html.r

```diff
@@ -0,0 +1,43 @@
+#' Returns HTML in a server.
+#'
+#' @param html_string character. A sring of HTML.
+#' @examples
+#' \dontrun{
+#'    routes <- list("/index" = function(...) html("<b>bold!</b>"))
```

Comment:
Hmm; good catch.  I would think that this is solved by (a) people being in the good habit of namespacing their functions (e.g., `microserver::html` vs. `rvest::html`) and (b) R warns you about when packages overwrite the functions of other packages on load.